src/analysis/vector/geometry_checker/qgsgeometrycheckerutils.h

```diff
@@ -110,6 +124,13 @@ class ANALYSIS_EXPORT QgsGeometryCheckerUtils
                        const QList<QgsWkbTypes::GeometryType> &geometryTypes,
                        const QgsGeometryCheckContext *context );
 
+        /**
+         * \ingroup analysis
+         *
+         * An iterator over all features in a QgsGeometryCheckerUtils::LayerFeatures.
+         * 
+         * \since QGIS 3.4
+         */
         class iterator
```

Comment:
Why this this class identifier does not begin with a capital letter?

Reply:
For libstd compatiblity I assume.